core-bundle/contao/templates/twig/backend/chrome/default.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends '@Contao/be_main.html5' %}
+
+{% block header %}
+    {{ parent() }}
+    {% include "@Contao/backend/component/message/_outlet.html.twig" %}
```

Comment:
Looks like this should go to the `be_main` itself?

Reply:
I'm not sure. I would move this to the `be_main` as soon as we need messages/dialogs everywhere. We then might want to enhance the `$this->insert()` function to allow including Twig templates. Or we find a way to completely replace be_main with Twig without breaking too much.

Reply:
Just saying, third party devs would have to extend from this "chrome" template now instead of `be_main` which feels weird.

Reply:
Changed in 024865647a819d82e36578d2ad5c0cacce4a5ad3